### PR TITLE
Fix mistake in theme style sample

### PR DESCRIPTION
**Pull request by ndsvw**
Removed line that overwrote "style.Outline", if it was set in the switch-case block.

Also improved code comments that did not match the code.

Did not create an issue, because it's a very small fix...

### CODE REVIEW

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs [hunk 1]
```diff
@@ -64,19 +64,19 @@ private static ThemeStyle CreateThemeStyle()
 
             switch (f["NAME"]?.ToString()?.ToLower())
             {
-                case "brazil": //If country name is Denmark, fill it with green
+                case "brazil": //If country name is Barzil, fill it with green
```
Comment (inforithmics) [inline Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:67]: small Typo should be Brazil

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs [hunk 2]
```diff
@@ -64,19 +64,19 @@ private static ThemeStyle CreateThemeStyle()
...
                     style.Fill = new Brush(Color.Green);
                     style.Outline = new Pen(Color.Black);
                     break;
-                case "united states": //If country name is USA, fill it with Blue and add a red outline
+                case "united states": //If country name is USA, fill it with violet
                     style.Fill = new Brush(Color.Violet);
                     style.Outline = new Pen(Color.Black);
                     break;
-                case "china": //If country name is China, fill it with red
+                case "china": //If country name is China, fill it with orange
```
Comment (inforithmics) [inline Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:75]: what is the reason for choosing the color orange for china

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs [hunk 3]
```diff
@@ -64,19 +64,19 @@ private static ThemeStyle CreateThemeStyle()
...
                     style.Fill = new Brush(Color.Orange);
                     style.Outline = new Pen(Color.Black);
                     break;
-                case "japan": //If country name is China, fill it with red
+                case "japan": //If country name is Japan, fill it with cyan
```
Comment (inforithmics) [inline Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:79]: What is the reason for choosing the color cyan for Japan

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs [hunk 4]
```diff
@@ -64,19 +64,19 @@ private static ThemeStyle CreateThemeStyle()
 
             switch (f["NAME"]?.ToString()?.ToLower())
             {
-                case "brazil": //If country name is Denmark, fill it with green
+                case "brazil": //If country name is Barzil, fill it with green
                     style.Fill = new Brush(Color.Green);
                     style.Outline = new Pen(Color.Black);
                     break;
-                case "united states": //If country name is USA, fill it with Blue and add a red outline
+                case "united states": //If country name is USA, fill it with violet
```
Comment (inforithmics) [inline Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:71]: what is the reason for choosing the color violet for United States

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs [hunk 5]
```diff
@@ -85,7 +85,6 @@ private static ThemeStyle CreateThemeStyle()
                     style.Outline = new Pen(Color.FromArgb(0, 64, 64, 64));
                     break;
             }
-            style.Outline = new Pen(Color.FromArgb(0, 64, 64, 64));
```
Comment (inforithmics) [inline Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:88]: good catch

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs (hunk 1 above)
Comment (ndsvw) [reply Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs]: yes. fixed it.

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs (hunk 2 above)
Comment (ndsvw) [reply Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:75]: Hi, I did not change the color. I made the comment match what the C# code in the 2 lines below is doing.

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs (hunk 3 above)
Comment (ndsvw) [reply Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:79]: Hi, I did not change the color. I made the comment match what the C# code in the 2 lines below is doing.

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs (hunk 4 above)
Comment (ndsvw) [reply Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:71]: Hi, I did not change the color. I made the comment match what the C# code in the 2 lines below is doing.

File: Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs (hunk 5 above)
Comment (ndsvw) [reply Samples/Mapsui.Samples.Common/Maps/Styles/ThemeStyleSample.cs:88]: thank you

**Review by inforithmics — APPROVED** [review]
Thank you, for this pull request

**Review by pauldendulk — APPROVED** [review]
Thanks for the fix @ndsvw , Thanks for the review @inforithmics

**Pull request merged**